src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Be careful, the bulldozer  started !!!");
```

Comment:
The message printed should be more informative and indicate that it is a Bulldozer that is working. Consider changing the message to include 'Bulldozer' explicitly, for example: 'Bulldozer is starting its work!'